go/worker/common/committee/group.go

```diff
@@ -567,11 +466,7 @@ func (g *Group) HandlePeerMessage(unusedPeerID signature.PublicKey, msg *p2p.Mes
 	return g.handler.HandlePeerMessage(ctx, msg)
 }
 
-func (g *Group) publishLocked(
-	spanCtx opentracing.SpanContext,
-	ci *CommitteeInfo,
```

Comment:
why was this already unused? where have we been sending these messages?

Reply:
Yeah it was unused since we migrated to gossipsub. Messages are published to all P2P peers subscribing to the runtime-specific topic via gossipsub.